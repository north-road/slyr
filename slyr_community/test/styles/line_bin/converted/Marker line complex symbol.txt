[{'class': 'QgsMarkerLineSymbolLayer',
  'interval': '2',
  'interval_unit': 'Point',
  'offset_along_line': '0.5',
  'offset_along_line_unit': 'Point',
  'offset_unit': 'Point',
  'subsymbol': [{'class': 'QgsSimpleMarkerSymbolLayer',
                 'color': '0,0,255,255',
                 'offset': '3,-7',
                 'offset_unit': 'Point',
                 'outline_style': 'no',
                 'size': '6',
                 'size_unit': 'Point'},
                {'angle': '-90',
                 'class': 'QgsSimpleMarkerSymbolLayer',
                 'offset': '-3,-5',
                 'offset_unit': 'Point',
                 'outline_style': 'no',
                 'size': '8',
                 'size_unit': 'Point'}]}]
