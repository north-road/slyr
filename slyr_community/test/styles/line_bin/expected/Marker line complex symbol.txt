{'levels': [{'color': None,
             'decoration': None,
             'decoration_on_top': False,
             'enabled': True,
             'flip': False,
             'join': 'miter',
             'line_start_offset': 0.0,
             'locked': False,
             'miter_limit': 10.0,
             'offset': 0.0,
             'pattern_marker': {'halo': False,
                                'halo_size': 2.0,
                                'halo_symbol': {'color': {'B': 255,
                                                          'G': 255,
                                                          'R': 255,
                                                          'dither': True,
                                                          'is_null': False,
                                                          'stream_offset': '0x115',
                                                          'type': 'RgbColor',
                                                          'version': 1},
                                                'enabled': True,
                                                'fill_style': 'solid',
                                                'locked': False,
                                                'outline': {'color': {'B': 110,
                                                                      'G': 110,
                                                                      'R': 110,
                                                                      'dither': True,
                                                                      'is_null': False,
                                                                      'stream_offset': '0xd2',
                                                                      'type': 'RgbColor',
                                                                      'version': 1},
                                                            'enabled': True,
                                                            'line_type': 'solid',
                                                            'locked': False,
                                                            'stream_offset': '0xc0',
                                                            'symbol_level': 0,
                                                            'type': 'SimpleLineSymbol',
                                                            'version': 1,
                                                            'width': 0.0},
                                                'stream_offset': '0xae',
                                                'symbol_level': 0,
                                                'type': 'SimpleFillSymbol',
                                                'version': 1},
                                'levels': [{'angle': 0.0,
                                            'color': {'B': 255,
                                                      'G': 0,
                                                      'R': 0,
                                                      'dither': True,
                                                      'is_null': False,
                                                      'stream_offset': '0x166',
                                                      'type': 'RgbColor',
                                                      'version': 1},
                                            'enabled': True,
                                            'locked': True,
                                            'marker_type': 'circle',
                                            'outline_color': {'B': 0,
                                                              'G': 0,
                                                              'R': 0,
                                                              'dither': False,
                                                              'is_null': False,
                                                              'stream_offset': '0x1ca',
                                                              'type': 'RgbColor',
                                                              'version': 1},
                                            'outline_enabled': False,
                                            'outline_size': 1.0,
                                            'rotate_with_transform': True,
                                            'size': 6.0,
                                            'stream_offset': '0x154',
                                            'symbol_level': 0,
                                            'type': 'SimpleMarkerSymbol',
                                            'version': 2,
                                            'x_offset': 3.0,
                                            'y_offset': 7.0},
                                           {'angle': 0.0,
                                            'color': {'B': 0,
                                                      'G': 0,
                                                      'R': 255,
                                                      'dither': True,
                                                      'is_null': False,
                                                      'stream_offset': '0x20d',
                                                      'type': 'RgbColor',
                                                      'version': 1},
                                            'enabled': True,
                                            'locked': True,
                                            'marker_type': 'circle',
                                            'outline_color': {'B': 0,
                                                              'G': 0,
                                                              'R': 0,
                                                              'dither': False,
                                                              'is_null': False,
                                                              'stream_offset': '0x271',
                                                              'type': 'RgbColor',
                                                              'version': 1},
                                            'outline_enabled': False,
                                            'outline_size': 1.0,
                                            'rotate_with_transform': True,
                                            'size': 8.0,
                                            'stream_offset': '0x1fb',
                                            'symbol_level': 0,
                                            'type': 'SimpleMarkerSymbol',
                                            'version': 2,
                                            'x_offset': -3.0,
                                            'y_offset': 5.0}],
                                'stream_offset': '0x39',
                                'symbol_level': 0,
                                'type': 'MultiLayerMarkerSymbol',
                                'version': 3},
             'stream_offset': '0x1e',
             'symbol_level': 0,
             'template': {'pattern_interval': 1.0,
                          'pattern_parts': [[1.0, 1.0]],
                          'stream_offset': '0x2ce',
                          'type': 'LineTemplate',
                          'version': 1},
             'type': 'MarkerLineSymbol',
             'version': 2}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerLineSymbol',
 'version': 2}
